gui/WiiLoadDialog.py

```diff
@@ -0,0 +1,110 @@
+import logging
+
+from PySide6 import QtCore
+from PySide6.QtCore import QSize, QStorageInfo, QDir, Qt, QEvent
+from PySide6.QtGui import QIcon, QGuiApplication, QFont
+from PySide6.QtWidgets import QDialog, QDialogButtonBox, QListWidgetItem, QMessageBox, QPlainTextEdit
+
+import gui_helpers
+import wiiload
+from gui import ui_WiiLoadDialog
+from utils import resource_path, file_size
+import serial
+import serial.tools.list_ports
+
+class WiiLoadDialog(ui_WiiLoadDialog.Ui_Dialog,QDialog):
+    def __init__(self, package, parent=None):
+        super().__init__(parent)
+        self.setupUi(self)
+        
+        self.setFont(QFont('Helvetica Neue'))
```

Comment:
This single setFont function slows down dialog initialization by 3845%. Might be because I'm running this on my secondary drive. Anyways, probably can live without it.
```suggestion
```

Reply:
Ironically, there was a delay if setFont wasn't called on my Mac, but I think that was due to a bad install of pyside6. I'll remove it.